tests/py/test_payment_instruction_api.py

```diff
@@ -0,0 +1,199 @@
+import json
+
+from gratipay.testing import Harness
+
+class TestPaymentInstructionApi(Harness):
+
+    def test_get_with_team_filter(self):
+        "Test that GET with team_slug passed returns correct response."
+
+        alice = self.make_participant("alice", claimed_time='now')
+        gratipay = self.make_team("gratipay", is_approved=True)
+        gittip = self.make_team("gittip", is_approved=True)
```

Comment:
A blast from the past! :-)

In other parts of the test suite we use `Enterprise` and `Trident` as example Teams.